src/autogluon_dashboard/plotting/framework_boxplot.py

```diff
@@ -0,0 +1,40 @@
+from typing import Union
+
+import hvplot
+
+from .all_plots import Plot
+
+
+class FrameworkBoxPlot(Plot):
+    def __init__(
+        self,
+        plot_title: str,
+        dataset_to_plot: hvplot.Interactive,
+        plot_type: str = "hvplot",
+        x_axis: Union[str, list] = None,
+        y_axis: Union[str, list] = None,
+        graph_type: str = "box",
+        xlabel: str = "",
+        ylabel: str = "",
+        label_rot: int = 90,
+        table_cols: list = ...,
```

Comment:
Is this a correct syntax by providing default list as `...`?

Reply:
I could do empty list to be more specific